ocaml/gencert/selfcert.mli

```diff
@@ -0,0 +1,9 @@
+val host : string -> string list -> string -> unit
+(** [certify hostname alt_names path] creates (atomically) a PEM file at
+[path] for hostname with alternative names [alt_names]. *)
+
+module CLI : sig
+  val main : unit -> unit
```

Comment:
I expected to see this binary installed in the Makefile

Reply:
This is not installed. I could remove it as I'm not building the stand-alone binary.